### PR TITLE
Update newton-dynamics: fix compile error in Newton.h

### PR DESCRIPTION
**Pull request by ImperatorS79**
This fix has been committed in newton main repository but no new version yet.

### CODE REVIEW

**Comment by Biswa96** [comment]
As this change is a upstream commit, a comment with that commit link in PKGBUILD file would be useful to maintain this patch.

**Comment by ImperatorS79** [comment]
I have no idea which commit it is unfortunately. The history is broken because the folder structure was changed and I do not know how to search for that in github.

**Comment by Biswa96** [comment]
Alright. The repository seems to be weird. Here is the commit https://github.com/MADEAPPS/newton-dynamics/commit/a6782a4c50c68964d20ca8ebf9621da14bf8ebf0

**Comment by ImperatorS79** [comment]
Obviousy the sha256sum 🤦.

File: mingw-w64-newton-dynamics/PKGBUILD [hunk 1]
```diff
@@ -56,6 +60,10 @@ build() {
     -DCMAKE_BUILD_TYPE="Release"\
     -G"MSYS Makefiles" \
     -DNEWTON_DOUBLE_PRECISION=ON \
+    -DNEWTON_WITH_SSE4_PLUGIN=OFF \
+    -DNEWTON_WITH_AVX2_PLUGIN=OFF \
+    -DNEWTON_WITH_AVX_PLUGIN=ON \
+    -DNEWTON_WITH_REFERENCE_GPU_PLUGIN=ON \
```
Comment (lazka) [inline mingw-w64-newton-dynamics/PKGBUILD:66]: Why these changes?

Comment (ImperatorS79) [reply mingw-w64-newton-dynamics/PKGBUILD:66]: I tried to enable avx2 and sse4 plugin but it failed and I forgot those modifications in the PKGBUILD.

Comment (lazka) [reply mingw-w64-newton-dynamics/PKGBUILD:66]: ok, I see they are the default anyway, thanks
